drivers/hmis/app/graphql/types/hmis_schema/external_form_submission.rb

```diff
@@ -22,8 +22,13 @@ class HmisSchema::ExternalFormSubmission < Types::BaseObject
     custom_data_elements_field
 
     def custom_data_elements
-      cdeds = load_ar_association(definition, :custom_data_element_definitions, scope: Hmis::Hud::CustomDataElementDefinition.order(:key))
-      resolve_custom_data_elements(object, definition_scope: cdeds)
+      load_ar_association(
```

Comment:
I think this isn't correct and we want to still be using `resolve_custom_data_elements` here, but I was having some trouble getting that to work, since `resolve_custom_data_elements` no longer accepts a `definition_scope` and expects the `record` passed in to have a `custom_data_elements` association, which this form submission object (I think) doesn't have. I will keep working on this to fix